sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/SonarAnalyzer.UnitTest.csproj

```diff
@@ -121,4 +121,10 @@
     </ProjectReference>
   </ItemGroup>
 
+  <ItemGroup>
+    <Reference Include="SonarAnalyzer.CFG">
+      <HintPath>..\..\src\SonarAnalyzer.CFG\bin\Debug\net46\SonarAnalyzer.CFG.dll</HintPath>
+    </Reference>
+  </ItemGroup>
+

```

Comment:
Can't we simply use `ProjectReference` here, as it is done for the others, instead of the direct dll reference ?